libs/application/templates/financial-aid/src/fields/Logo/Logo.css.ts

```diff
@@ -0,0 +1,11 @@
+import { style } from '@vanilla-extract/css'
+import { theme } from '@island.is/island-ui/theme'
+
+export const logo = style({
+  width: '50%',
+  '@media': {
+    [`screen and (min-width: ${theme.breakpoints.md}px)`]: {
+      width: '33%',
+    },
+  },
+})
```

Comment:
the styles on the logo are weird imo
Why is it not in the middle of the screen?

Reply:
1. They want the logo to be on every screen
2. They want the logo to be right aligned after we showed them other versions